k8s/frontend-deployment.yml

```
@@ -0,0 +1,27 @@
+  namespace: chat-app
+
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: frontend
+  template:
+    metadata:
+      name: frontend-pod
+      namespace: chat-app
+      labels: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)